packages/material-ui/src/RadioGroup/RadioGroup.test.js

```diff
@@ -1,170 +1,185 @@
 import React from 'react';
 import { assert } from 'chai';
 import { spy } from 'sinon';
-import { createShallow, createMount } from '@material-ui/core/test-utils';
+import { act } from 'react-dom/test-utils';
+import { createMount, findOutermostIntrinsic, testRef } from '@material-ui/core/test-utils';
 import FormGroup from '../FormGroup';
 import Radio from '../Radio';
 import RadioGroup from './RadioGroup';
 import consoleErrorMock from 'test/utils/consoleErrorMock';
 
 describe('<RadioGroup />', () => {
-  let shallow;
   let mount;
 
   before(() => {
     mount = createMount();
-    shallow = createShallow();
   });
 
   after(() => {
     mount.cleanUp();
   });
 
+  function findRadio(wrapper, value) {
+    return wrapper.find(`SwitchBase[value="${value}"]`).first();
+  }
+
+  function findRadioInput(wrapper, value) {
+    return wrapper
+      .find(`SwitchBase[value="${value}"]`)
```

Comment:
Is the value not applied to the native `input`?

Reply:
Good point, it should be. I’ll check 😅